docs/customization.rst

```diff
@@ -1,11 +1,36 @@
 =====================
-Customizing ``xonsh``
+Updating and customizing xonsh
 =====================
 
 .. contents::
    :local:
 
-How do I...
+Updating xonsh
+===========
```

Comment:
same here

Reply:
Ahhh. I missed that. Maybe we should have our docs build fail if there are warnings.

Reply:
Probably a good idea to let the build fail if you want to enforce this (I don't think the RST format needs it, but I might be wrong).

Reply:
rest definitely needs this, and we do have a doc test on Travis, so I think this might just be a warning that renders incorrectly.  Maybe there is a way to turn that into a hard failure. Thanks again!